### ISSUE TITLE
GameWindow.CursorVisible doesn't work on Linux (X11) 

### ISSUE THREAD

**Issue by Sioma112233** [opentk/opentk#1102]
### Description
Hi, was using OpenTK 3.2.0 from Nuget, on linux (with X11, didn't try wayland)
And setting `GameWindow.CursorVisible` to `false` does nothing and it also seems setting `GameWindow.CursorGrabbed` to `true` doesn't do anything either.

I tried downgrading the version of OpenTK, and the issue persist also on 3.1.0, but in 3.0.1 it works perfectly.
Though, 3.0.1 doesn't have `CursorGrabbed` so I can't test it.

### Repro steps

1. Use some linux distro with a DE/WM using X11 (not wayland)
2. Create a new C# project, reference OpenTK version 3.2.0
3. Create a window and set `CurosrVisible` to `false`
``` csharp
public static void Main()
{
	var window = new GameWindow();
	window.CursorVisible = false;

	window.Run();
}
```
4. Run your project

### Expected behavior
A window appears and your cursor disappears

### Actual behavior
A window appears but your cursor is still visible

### Related information 

* OS: Fedora 32, Gnome DE on X11
* OpenTK versions: 3.2.0, 3.1.0
* Runtime: Mono 6.10.0.104 (TargetFramework=4.8)

**Comment by NogginBops** [comment]
I'm pretty sure that this is because there is no window so you can't modify the cursor. 
You have to call `window.Run()` or any equivalent function the actually realizes the window before changing the cursor (typically done in the `OnLoad` function).

**Comment by Sioma112233** [comment]
unfortunetly, changing the window's `CursorVisible` property inside `OnLoad` also doesn't work

**Comment by Sioma112233** [comment]
for example:
``` csharp
var window = new GameWindow(800, 800);
window.Load += (_, __) => window.CursorVisible = false;
			
window.Run();

```

**Comment by jvbsl** [comment]
Have you tried explicitely setting it to true and setting it to false after that? Because in the source code there is a check whether the value is different to the old value.

**Comment by Sioma112233** [comment]
tried now, 
1. 1 attempt to set it to `true` before `OnLoad` (where it is set to false)
2. 1 attempt to set in in `OnLoad`, right before setting it to false

both failed

**Comment by jvbsl** [comment]
FocusedChanged worked for me, you can introduce a bool variable to only do it once at the beginning, seems to work for SDL as well as X11

**Comment by Sioma112233** [comment]
Yes, setting `CursorVisible` in a `FocusedChanged' event handler works.
Seems that part of what happens when focus changes is that the `CursorVisible` property gets set to true always (from debugging, this is true at-least for Sdl2NativeWindow's "CursorVisible" property)

So it makes sense that the "FocusedChanged += " solution works as we "re-set" it to false after it is being set to true.

**Comment by varon** [comment]
should be closed thanks to @leezer3  in #1178
